### ISSUE TITLE
Crash Click Palette Demo Window

### ISSUE THREAD

**Issue by pbc37** [ocornut/imgui#2320]
```
Dear ImGui 1.67 WIP (16602)
--------------------------------
sizeof(size_t): 8, sizeof(ImDrawIdx): 2, sizeof(ImDrawVert): 20
define: __cplusplus=201500
define: _WIN32
define: _WIN64
define: __MINGW32__
define: __MINGW64__
define: __GNUC__=6
--------------------------------
io.BackendPlatformName: imgui_impl_glfw
io.BackendRendererName: imgui_impl_opengl3
io.ConfigFlags: 0x00000000
io.ConfigInputTextCursorBlink
io.ConfigWindowsResizeFromEdges
io.BackendFlags: 0x00000006
 HasMouseCursors
 HasSetMousePos
--------------------------------
io.Fonts: 1 fonts, Flags: 0x00000000, TexSize: 512,64
io.DisplaySize: 1280.00,720.00
--------------------------------
style.WindowPadding: 8.00,8.00
style.WindowBorderSize: 1.00
style.FramePadding: 4.00,3.00
style.FrameRounding: 0.00
style.FrameBorderSize: 0.00
style.ItemSpacing: 8.00,4.00
style.ItemInnerSpacing: 4.00,4.00

Operating System: Windows 7 64-bits Service Pack 1
Branch: Master
```

**My Issue/Question:**

When I click randomly at the Palette Window in the Demo Window it crashes.

**Screenshots/Video**

Console:
https://gyazo.com/4e103f97f44613afbbab7cd05156fe27

GIF:
https://gyazo.com/32f7c79e8a297cbe7421c05a4118c889

```
I did not change any code from the GLFW+OPENGL example
```

**Comment by ocornut** [comment]
Hello,

I cannot seem to reproduce your issue.
Could you provide a call-stack? (as requested by the Issue template?)

You say "I did not change any code from the GLFW+OPENGL example"
However your screenshot shows a path of "tfm_seriousgame/bin/main_win.exe" which is not the filename of the example, so this is contradictory.

Can you repro in the unmodified examples/ application?

(PS: You can attach pictures/GIF in your github posts, prefer using github attachment over Gyazo)

**Comment by ocornut** [comment]
Thanks for the callstack.
I think this bug existed for a few commits before 1.67 and then was fixed. Try updating your copy.

**Comment by ocornut** [comment]
I confirmed this bug was introduced by d845135 and fixed in b3469fa94b79ad56e978130cd28f863f1cbe7217 (Fixed on January 2nd). Sorry for the issue.

**Comment by pbc37** [comment]
Don't worry! I like imgui so I just wanted to help. Good luck!
________________________________
